### ISSUE TITLE
[CAIA Intake] Benefits Decision Review: HLR (VA-20-0996) form updates

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#82516]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name**: Benefits Decision Reviews
- **OCTO product owner:** @amylai-va 
- **Product name:** Higher Level Review
- **Product manager:**  @saderagsdale 
- **Slack channel:** [#benefits-decision-reviews](https://dsva.slack.com/archives/C5AGLBNRK)
- **Dedicated content writer on your team (if you have one):** 
- **Dedicated a11y specialist on your team (if you have one):** @davidakennedy 
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [x] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

`Tell us briefly about your work and the outcomes you’re aiming to achieve.`

The HLR paper form has been updated, and we want to get the digital form to match.

## Collaboration timeframe
**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

We are starting the work this upcoming sprint (5/8-5/21), and Benefits content team mentioned they'd like to embed with teams (if possible), so we're asking early for possible collaboration.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [x] I’m not sure

Forms expire within the year of the paper form update (updated published 4/30/2024), so this needs to be done within the year.

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

Form is already in production, we are updating a few of the screens to match form language. There is no major feature change, but there will be content changes.

### Will you release this new product incrementally (for example, release to 25% of users to start)?
- [x]  Yes
- [ ]  No
 
 *Note: **If you check yes, we’ll reach out to discuss details about the content in the react widget. We use these widgets to display entry points for new products to a certain percentage of users who visit our static pages. Please refer to this [GitHub reference](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md) for dynamic content.
 
- [ ]  I acknowledge that if I change the launch to be incremental, or change it from incremental to 100%, I must notify CAIA of this change as there is additional work involved that may impact deadlines.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** [90188](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90188)

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [FIGMA](https://www.figma.com/file/OxukHeNtSkCDbdmLxD5k9A/Higher-Level-Review-(VA-0996)?type=design&node-id=0%3A1&mode=design&t=UXbQ55kq5dfnizv7-1)
- [Content Source of Truth] - We will be using the Figma file as the content source of truth 
- [HLR Paper Form Final Changes](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/Shared%20Documents/Decision%20Reviews/higher%20level%20review/VAF20-0996%20mock-up%20FINAL%20w%20ALL%20CHANGES.docx?d=wf980329cb8bc4113b24d2f201e19e4cd&csf=1&web=1&e=s6MuVk)
- [HLR Paper Form](http://www.vba.va.gov/pubs/forms/VBA-20-0996-ARE.pdf)

- [SME Form Audit Meeting Pt 1](https://dvagov-my.sharepoint.com/:v:/g/personal/sade_ragsdale_va_gov/EXijpAC72E9NmEuMzof-3LMBcLGSvE1klW53Rq8LTL-Gdg?referrer=Teams.TEAMS-ELECTRON&referrerScenario=MeetingChicletGetLink.view.view)
- [MURAL](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1714761005965/cf4071f13c521f01c515757de3ab6e3e87ba22ba?sender=u80fb5a3b8f7951f35d701197)
## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`, and we’ll then acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [x] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

```[tasklist]
### A11Y Tasks
- [x] Sara/DK/EB to look into if PDF needs reviewed by CAIA A11Ys [NO REVIEW NEEDED AT THIS TIME (as of 5/29/2024)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82516#issuecomment-2145522006)
- [x] Priority flag added with Martha on 5/28/2024 call `Medium`
- [ ] Attend Kickoff Meeting 6/5/2024 1:30 p.m. ET `Sara & Evan`
```

```[tasklist]
### Content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/85174
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/86215
```

**Comment by eileen-coforma** [comment]
@strelichl Our team had a chat with Danielle, Beth, and Katherine a while back (in March) about how to better work with content on form updates (since updates can affect both unauthorized pages, as well as, our application). They mentioned that early involvement with the team, especially since we've established contact with SMEs, will be helpful in making updates.

**Comment by jstrothman** [comment]
@strelichl In this case, I'm the Benefits Portfolio OCTO designer who's been working with Danielle & Beth on prioritization so they can plan for content resources in support of our initiatives. In the past, it's been especially helpful for large initiatives when we've been able to have a content writer embed with the team. I'm aware that this possibility is highly dependent on the various demands on CAIA's time.

**Comment by aprocik1** [comment]
Thanks @eileen-coforma and @jstrothman! Two more questions:
- When do you think it makes sense for us to review your proposed designs? I know you're working on this during the current sprint (5/8-5/21). Katherine and I are managing a number of other priorities right now, so I want to make sure we review at the right time.
- I see your proposed designs for steps 1, 2, and 3 from January and April. Are those the designs you'd like us to review? Or are there other steps or updated designs we should review instead?

**Comment by eileen-coforma** [comment]
@aprocik1 Our designs aren't ready for review at this point. We will likely have designs drafted by the end of this sprint. I started this ticket to bring awareness that this work is starting, and to offer a chance for the writer who'd be working with us to sit in on these initial requirements conversations with SMEs. 

Some of the form update content could impact the unauthenticated content page for [HLR](https://www.va.gov/decision-reviews/higher-level-review/). Like @jstrothman mentioned, we've been told by content writers that it's extremely helpful to have direct contact with the SMEs for these updates.

**Comment by bethpottsVADEPO** [comment]
At this stage, for content's awareness of the work and any challenges that may need to be addressed with content solutions in the course of this work, how about we start by having Eileen record the next several stakeholder meetings for content and share those recordings--can that link be shared right in the ticket? In the meantime, we'll continue to prioritize Benefits portfolio work at the OCTO level and then at the ticket level.

**Comment by eileen-coforma** [comment]
@strelichl Wednesday noon is busy for me and my backup, @skylerschain. We can ether do Wednesday 1PM ET or Thursday or Friday 12PM ET.

**Comment by kristinoletmuskat** [comment]
@eileen-coforma hey hey! Checking in on IA support for this. Are ya'll considering moving the form, or are we just making updates to the form itself? And do you think you will add new or change any sub-urls within the form?

**Comment by eileen-coforma** [comment]
@strelichl Yes, 1:30ET Wednesday would work for both of us. Thanks!

@kristinoletmuskat We will just be making updates to the form itself. There might be one new page, but we still need to check with stakeholders. We don't foresee changing sub-urls within the form. On the other hand, we have a separate ticket for [SC (VA 20-0995)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83316), which will have new pages and sub-urls.

**Comment by kristinoletmuskat** [comment]
ok great! I'm going to untag IA here and look at that other one.

**Comment by sara-amanda** [comment]
## PDF Accessibility

On 5/29/2024, Sara and Evan spoke to DK about the PDFs re: ticket comment: 
> “The HLR paper form has been updated, and we want to get the digital form to match.”

- DK looked into how the forms are made. 
- **PDFs are not currently accessible.** 
    - PDFs are generated by LIghthouse. 
        - **Ticket**: Lighthouse is aware and has a ticket in play to work on a fix. 
        - **Issue**: Lighthouse uses Ruby as backend language, and there are not any accessible PDF libraries. 
        - **Workaround**: They explored a workaround that involves bringing in a  JAVA library to do it - but it is a heavy lift. 

*This has been added to the Kick-Off Meeting Notes for 6/5/2024.

**Comment by skylerschain** [comment]
[Here's the link to the Mural board](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1714761005965/cf4071f13c521f01c515757de3ab6e3e87ba22ba?sender=u80fb5a3b8f7951f35d701197). Red stickies contain links to Figma file where content is being updated.

**Comment by eileen-coforma** [comment]
@katherine-fung @aprocik1 Please let me know when your suggestions are finalized.

**Comment by katherine-fung** [comment]
Hi @eileen-coforma, 

We've provided our content recommendations in this Google doc: 
https://docs.google.com/document/d/1OZcdssks9WrtSCLK60ywdXxpOy-lautYuVcFlADNRhU/edit 

(We needed a space outside of Figma to go back and forth on edits. We'll let you and your team reconcile the content in Figma). 

Let us know if you have any questions!

**Comment by eileen-coforma** [comment]
@strelichl Can I get more context about this widget work? We're prepping for staging review right now, and will be ready to release once that is complete.

**Comment by eileen-coforma** [comment]
@strelichl Thanks for the clarification! I'm not sure if this is necessary for our two forms. They're both already launched in production, so we already have a React widget. All of our changes are happening within the form itself, and the form will remain live.

**Comment by eileen-coforma** [comment]
@strelichl I don't believe so. We only have suggested content changes for things within the application (which includes the [form introduction](https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/introduction)), but nothing that should affect the [static sitewide page](https://staging.va.gov/decision-reviews/higher-level-review/).

**Comment by eileen-coforma** [comment]
@strelichl I confirmed with our dev. Even with our incremental release, we won't need to use a React widget. We haven't been using React widgets for our forms. Folks coming in will either see the old content or new content within the form. This is for both of our forms.

**Comment by eileen-coforma** [comment]
@strelichl For HLR, they can continue the version. The data passed over should be the exact same.

**Comment by eileen-coforma** [comment]
@katherine-fung I sent the content to our stakeholders and got some feedback from Shireen. 
There are some pieces of feedback that seem to have been given a bit out-of-context, ie:
- Claims shouldn't be changed to decision reviews for the subtask page
- Intro page should link to the sitewide HLR page, and not directly to a PDF form

But there are also feedback that pertain to their suggestion vs sitewide style choices, ie:
- disability claims vs compensation claims

Can you take a look through this feedback and let us know what you suggest we accept/decline/inquire more about?

[Higher-Level Review Product Walkthrough Transcript.docx](https://github.com/user-attachments/files/16630370/Higher-Level.Review.Product.Walkthrough.Transcript.docx)

**Comment by katherine-fung** [comment]
Thanks @eileen-coforma! I've replied to Shireen's comments, and will continue the conversation with her.

**Comment by katherine-fung** [comment]
Hi @eileen-coforma, 

I've left comments in the Figma file that reflect Shireen's feedback from the PDF she emailed back to us last Thursday. 

I've indicated in Figma where we can go ahead and make the changes she's requested. Let me know if you want me to go in and change that text in Figma. 

Otherwise, I have 3 notes/questions: 

1.  I emailed Shireen about a way to her incorporate on the screen "Option to request an informal conference," and CC'ed you. 

2. Shireen left comments about the screen where the form asks Veterans to opt in to the modernized decision review process. Can you take a look at her comments in the PDF? She offered a way to show the plain language version of the policy, but I'm not sure if it's feasible.  

3. Shireen requested a copy change to the confirmation page. I realized that I didn't review the confirmation page in my original review (in the Google doc of my recommendations). I've left a few other recommendations for this screen via Figma comments, but I'm not sure if they're in scope. These recommendations are to align with confirmation page style that CAIA has been implementing in online forms. 

cc @aprocik1 @strelichl

**Comment by katherine-fung** [comment]
Closing this ticket because I've finished providing content recs for the HLR updates. 

Noting that for the questions in my last comment: 

1. Shireen approved the recommendation, and we implemented it. 

2. "Withdrawal from the legacy appeals process" screen implements Shireen's requests. 

3. Product team has implemented content recs for the confirmation screen. 

I'm also documenting that at this time, the homelessness question will not include hint text about why VA is asking the question. 

Per Eileen, "We’ve tested the homelessness question once before, and when prompted, people understood the need for a homelessness question and the demographic it’s targeting, even when it didn’t apply to them. We have not tested the question with participants experiencing homelessness, so we’d have to turn to other teams who have done that research.
We’ll be testing this question (with the more extended prioritization variation) on the SC research we’re doing next week."
